student-store-ui/src/components/App/App.jsx

```diff
@@ -1,21 +1,133 @@
 import * as React from "react"
-import { BrowserRouter } from 'react-router-dom'
+import { useEffect, useState } from "react"
+import axios from "axios"
+
+
+
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
 import Navbar from "../Navbar/Navbar"
 import Sidebar from "../Sidebar/Sidebar"
 import Home from "../Home/Home"
 import "./App.css"
+import Hero from "../Hero/Hero"
+import About from "../About/About"
+import Contact from "../Contact/Contact"
+import Footer from "../Footer/Footer"
+import ProductDetails from "../ProductDetails/ProductDetails"
 
 export default function App() {
+
+  const URL = "http://localhost:3001";
+
+  const [products, setProducts] = useState();
```

Comment:
you can create and store `products` state in the Home component, to avoid a layer of prop drilling